cargo-dist-schema/src/snapshots/cargo_dist_schema__emit.snap

```diff
@@ -324,6 +335,19 @@ expression: json_schema
           }
         }
       }
+    },
+    "SystemInfo": {
+      "description": "Info about the system/toolchain used to build this announcement.\n\nNote that this is info from the machine that generated this file, which *ideally* should be similar to the machines that built all the artifacts, but we can't guarantee that.\n\ndist-manifest.json is by default generated at the start of the build process, and typically on a linux machine because that's usually the fastest/cheapest part of CI infra.",
+      "type": "object",
+      "properties": {
+        "cargo_version_line": {
+          "description": "The version of Cargo used (first line of cargo -vV)\n\nNote that this is the version used on the machine that generated this file, which presumably should be the same version used on all the machines that built all the artifacts, but maybe not! It's more likely to be correct if rust-toolchain.toml is used with a specific pinned version.",
```

Comment:
"It's more likely" -> if rust-toolchain.toml is present and uses a specific version it will always be correct, right? the only way it wouldn't be correct is if rust-toolchain.toml wasn't present or if it was but specified something like "latest"?

Reply:
per https://rust-lang.github.io/rustup/overrides.html, several things have precedence over rust-toolchain.toml (like RUSTUP_TOOLCHAIN and `rustup override`), and I can't *guarantee* you haven't messed up your build system by using one of these in addition to rust-toolchain.toml.